src/hotspot/share/cds/cppVtables.cpp

```diff
@@ -66,19 +66,20 @@
 
 class CppVtableInfo {
   intptr_t _vtable_size;
-  intptr_t _cloned_vtable[1];
+  intptr_t _cloned_vtable[1]; // Pseudo flexible array member.
+  static size_t cloned_vtable_offs() { return offset_of(CppVtableInfo, _cloned_vtable); }
```

Comment:
I'd really prefer spelling out "offset" rather than saving two characters with the "offs" abbreviation.

Reply:
Sure. I don't mind.